omod/src/main/webapp/resources/scripts/sync2.conflict.js

```diff
@@ -0,0 +1,73 @@
+function conflictResolution() {
+
+    jq.getJSON('/' + OPENMRS_CONTEXT_PATH + '/sync2/AuditDetails/conflictResolution.action',
+    {
+        conflictLogUuid:$('#retryLogUuid').text()
+    }).success(function(data) {
+        window.location.replace('/' + OPENMRS_CONTEXT_PATH + data['url'])
+    });
+}
+
+function setValueOfRadioButton(key, customValue) {
+    document.getElementById(key).value = customValue.value;
+}
+
+function setValueOfJsonKey(obj, current, pathKey, pathValue) {
+    for(var key in obj) {
+        var value = obj[key];
+        var newKey = (current ? current + "." + key : key);
+        if(value && typeof value === "object") {
+            setValueOfJsonKey(value, newKey, pathKey, pathValue);
+        } else {
+            if(newKey == pathKey) {
+                obj[key] = pathValue;
+            }
```

Comment:
this method overrides its param please use deep copy
                   // deep copy set to true
                    var merged = jQuery.extend(true, {}, localObjJson);

Reply:
I think this will have better performance:
        var localObjJson = ${localObjJson};
        var foreignObjJson = ${foreignObjJson};
        var mergedObjJson = ${localObjJson}; //assign the same value as to localObjJson
Is this ok in your opinion?

Reply:
it seems to be the same object or shallow copy, will they be updated separately?

Reply:
Yes, they will be independent